test/utils/query.js

```diff
@@ -0,0 +1,181 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import query from '../../src/utils/query';
+
+describe('query utils', () => {
+	const defaultEndpoint = 'accounts';
+	const defaultParameter = {
+		address: 'address1',
+		limit: 1,
+	};
+	const defaultArrayParameters = [
+		{
+			address: 'address1',
+			limit: 1,
+		},
+		{
+			address: 'address2',
+			limit: 1,
+		},
+	];
+
+	let apiClient;
+	let response;
+	describe('when the response does not have data', () => {
+		beforeEach(() => {
+			response = {
+				no: 'data',
+			};
+			apiClient = {
+				accounts: {
+					get: sandbox.stub().resolves(response),
+				},
+			};
+			query(apiClient, defaultEndpoint, defaultParameter);
+			return Promise.resolve();
+		});
+
+		it('it should call API client', () => {
+			return expect(apiClient.accounts.get).to.be.calledWithExactly(
+				defaultParameter,
+			);
+		});
+
+		it('it should thrown an error', () => {
+			return expect(
+				query(apiClient, defaultEndpoint, defaultParameter),
```

Comment:
Why not store this in a `result` variable in the `beforeEach` hook, rather than calling `query` twice?